docs/content/integrations/airflow.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Dagster with Airflow | Dagster
-description: The dagster-airflow package allows you to export Dagster jobs as Airflow DAGs, as well as to import Airflow DAGs into Dagster jobs.
+description: The dagster-airflow package allows you to export Dagster jobs as Airflow DAGs, as well as trigger job runs from Airflow.
```

Comment:
I think this is wrong. We no longer export jobs as Airflow DAGs. But we can ingest Airflow DAGs and execute them natively in Dagster.